modules/auth/handle_file.go

```diff
@@ -39,21 +39,25 @@ func (m *Module) IsFileOpAuthorised(project, token, path string, op utils.FileOp
 
 func (m *Module) getFileRule(path string) (map[string]interface{}, *config.FileRule, error) {
 	pathParams := make(map[string]interface{})
+	ps := "/"
+	if m.fileStoreType == string(utils.Local) {
+		ps = string(os.PathSeparator)
+	}
 
 	// Check if its a valid absolute path
-	if strings.Contains(path, string(os.PathSeparator) + "..") {
+	if strings.Contains(path, ps + "..") {
```

Comment:
Let's check if it contains a `..` no need for the prefixed /

Reply:
Done